packages/gatsby-theme-carbon/src/components/Meta.js

```diff
@@ -14,6 +14,8 @@ const Meta = ({ pageTitle, pageDescription, pageKeywords, titleType }) => {
         return title;
       case 'append':
         return `${title}${pageTitle ? ` – ${pageTitle}` : ''}`;
+      case 'prepend':
+        return `${pageTitle}${pageTitle ? ` – ${title}` : ''}`;
```

Comment:
We probably want to flip this so in the event there’s no page title we still have a title. This is a factor for things like the homepage and 404. 

The hyphen should be attached to the pageTitle so we don’t have a dangling hyphen in that scenario as well.